### PR TITLE
Documentation tidy ups

### PR DESCRIPTION
**Pull request by jdyankova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,71 +98,70 @@ Default variables values are as follows:
...
+  rancher_secret_key = xyz1xyz2xyz
+  rancher_url = https://3.4.1.2
+  ```
+- Use `SECURE_SOURCE_IP` to add any IP to the monitoring services firewalls so Zabbix Web and Kibana UI can be reached
   from that IP
-- The Zabbix Web Frontend can be accessed on a browser using the public Zabbix Server IP. User: Admin
-- Kibana UI can be accessed on a browser using the public ELK Server IP and port 5601
+- The Zabbix Web Frontend can be accessed on a browser using the public Zabbix Server IP. User: `Admin`
+- Kibana UI can be accessed from a browser using the public ELK Server IP and port 5601
 - A multi-cloud setup on both AWS and GCP currently only allows the creation of Zabbix and Elastic Stack servers on AWS.
```
Comment (jdyankova) [inline README.md:153]: I was uncertain about the intent here. Checking the what the help command does (which is the section where these previously lived) does not do or relate to any of the above points, so suggesting this 'miscellaneous' section instead

File: README.md [hunk 2]
```diff
@@ -98,71 +98,70 @@ Default variables values are as follows:
...
 
-```
-./mcadm.sh destroy manager
-```
```
Comment (jdyankova) [inline README.md:154]: This seems completely unrelated to the items above it. Feels like an oversight / leftover. Happy to bring back otherwise.

File: README.md [hunk 3]
```diff
@@ -311,33 +310,11 @@ The `mcpadm` scripts can also get, deploy and destroy terraform infrastructure,
...
-  specs:
-    app1:
-      name: &name ecat-admin
-      runtime: custom
-      env: flex
-      service: *name
-```
+We recommend using [YAML anchors](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) to reduce duplication and ensure
+consistency between various configs. Description of this feature is available in a variety of external sources,
+e.g. [simple summary lives here](https://www.educative.io/blog/advanced-yaml-syntax-cheatsheet#anchors).
```
Comment (jdyankova) [inline README.md:315]: This is information about a widely documented format. In my opinion, it shouldn't be our job to try to re-explain fundamental concepts, so am proposing to reduce the amount of docs we maintain in this manner.

**Review by gbmeuk — APPROVED** [review]
